docs/source/theming/using-third-party-themes.md

```diff
@@ -0,0 +1,110 @@
+# Using third party libraries/themes different from semantic-ui
+
+You can use Volto with third party libraries or themes written in sass and avoid applying semantic-ui on public facing views.
+This is made possible by code splitting, where interfaces have a marker CSS class to incapsulate styles and avoid conflicts between semantic-ui and the custom theme you would use.
+
+## The problem
+
+The main purpuse could be to use a sass based theme like Bootstrap.
+If you want to load a different styling library using the base Volto configuration, you will load a huge bundle with both having weight and performance issues thus this would likely imply conflicts on base elements as containers.
```

Comment:
```suggestion
If you want to load a different styling library using the base Volto configuration, you will load a huge bundle with both having weight and performance issues thus this would likely imply conflicts on base elements like containers.
```